internal/git/git.go

```diff
@@ -353,7 +350,7 @@ func NumberCommits(sha1, sha2 string) int {
 	cmd.Stderr = nil
 	CmdOut, err := cmd.Output()
 	if err != nil {
-		fmt.Printf("There are no commits between %s and %s", sha2, sha2)
+		fmt.Printf("There are no commits between %s and %s", sha1, sha2)
```

Comment:
Just noticed this also needs a linebreak `\n` at the end of Printf() string.

Reply:
Good catch!